bwakit/Dockerfile

```diff
@@ -2,16 +2,20 @@ FROM ubuntu
 
 MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
 
-RUN apt-get update && \
-  apt-get install -y \
-    curl
+RUN apt-get update; apt-get install -y wget bzip2 perl
 
 # pull down bwa-kit
-RUN curl -L http://sourceforge.net/projects/bio-bwa/files/bwakit/bwakit-0.7.12_x64-linux.tar.bz2 \
-  | tar -xjC /opt/
+RUN wget http://sourceforge.net/projects/bio-bwa/files/bwakit/bwakit-0.7.12_x64-linux.tar.bz2
+
+RUN tar -jxvf bwakit-0.7.12_x64-linux.tar.bz2 -C /opt/
+
+RUN rm /opt/bwa.kit/run-bwamem; wget -O /opt/bwa.kit/run-bwamem https://raw.githubusercontent.com/BD2KGenomics/bwa/master/bwakit/run-bwamem; chown 500:500 /opt/bwa.kit/run-bwamem; chmod +x /opt/bwa.kit/run-bwamem
```

Comment:
Also, I would rather include the patched run-bwamem file here in the cgl-docker-lib repo and do an `ADD`/`COPY`. It's much easier to guarantee that a path in a repo will be unchanged, vs a URL to another repo.

I also think it is preferable because then anyone who is looking at this Dockerfile and who wants to see what the change we are making to the `run-bwamem` script has the file in their local checkout of this repo.